src/Functions/multiIf.cpp

```diff
@@ -254,8 +253,12 @@ class FunctionMultiIf final : public FunctionIfBase
     if (which.is##TYPE()) \
     { \
         MutableColumnPtr res = ColumnVector<TYPE>::create(rows); \
-        executeInstructionsColumnar<TYPE, INDEX>(instructions, rows, res); \
-        return std::move(res); \
+        MutableColumnPtr null_map = ColumnUInt8::create(rows); \
```

Comment:
Do we need to create `null_map` irregardless of `result_type->isNullable()`?

Reply:
done